config/config.go

```diff
@@ -120,16 +125,42 @@ func Load(filename string) (*Config, error) {
 		return nil, err
 	}
 
-	if err := Validate(&conf); err != nil {
+	if err := Validate(&conf, time.Now); err != nil {
 		return nil, err
 	}
 	return &conf, nil
 }
 
-func Validate(conf *Config) error {
+func Validate(conf *Config, getNow func() time.Time) error {
 	// Set any defaults.
-	if conf.ExportURL == "" {
-		conf.ExportURL = DefaultExportURL
+	if conf.ApiURL == "" {
+		conf.ApiURL = DefaultApiURL
+	}
+
+	// TODO: enforce lower limit?
+	if conf.ExportDuration.Duration == 0 {
+		if conf.GroupFilesByDay {
+			log.Println(`WARNING: The "GroupFilesByDay" option is deprecated. Please use "ExportDuration" instead.`)
+			conf.ExportDuration.Duration = 24 * time.Hour
+		} else {
+			log.Println(`INFO: "ExportDuration" not set in config. Defaulting to 1 hour`)
+			conf.ExportDuration.Duration = time.Hour
+		}
+	} else if (24*time.Hour)%conf.ExportDuration.Duration != 0 {
```

Comment:
This is a great question! We _should_ be using UTC for the for all of the times or at least converting them to UTC from the config file so that we don't have to worry about DST. I should make sure that's the case, though.

Reply:
Oh yeah, duh! UTC is fine here.